sdk/eventgrid/azure-eventgrid/tests/conftest.py

```diff
@@ -25,10 +25,15 @@
 # --------------------------------------------------------------------------
 import platform
 import sys
-
+import pytest
+from devtools_testutils.sanitizers import add_remove_header_sanitizer
 
 # Ignore async tests for Python < 3.5
 collect_ignore_glob = []
 if sys.version_info < (3, 5):
     collect_ignore_glob.append("*_async.py")
     collect_ignore_glob.append("test_cncf*")
+
+@pytest.fixture(scope="session", autouse=True)
+def add_aeg_sanitizer():
```

Comment:
Importing `test_proxy` from `devtools_testutils` and referencing the fixture here should allow tests to use the proxy in CI:
```suggestion
def add_aeg_sanitizer(test_proxy):
```